### PR TITLE
Added support ZTD feature

### PR DESCRIPTION
**Pull request by skg-net**
Signed-off-by: Senthil Kumar Ganesan <skg.net.dev@gmail.com>

### CODE REVIEW

File: release_bp/OPX_dell_base_stretch.xml [hunk 1]
```diff
@@ -80,7 +80,10 @@ blueprint files.
   <inst_hook>97-create-manifest.postinst.sh</inst_hook>
   <inst_hook>98-set-apt-sources.stretch.postinst.sh</inst_hook>
   <inst_hook>99-set-corefile-setting.postinst.sh</inst_hook>
+  <inst_hook>95-enable-ztd-mode.postinst.sh</inst_hook>
```
Comment (heucke) [inline release_bp/OPX_dell_base_stretch.xml]: Please add these in sorted order.

**Review by heucke — CHANGES_REQUESTED** [review]
The PR says "Added" ztd, but the contents of this PR seem to **enable** ztd. Please update the commit message if I am not mistaken.

File: release_bp/OPX_dell_base_stretch.xml [hunk 2]
```diff
@@ -80,7 +80,10 @@ blueprint files.
...
+  <inst_hook>96-setup-ztd-dhclient.postinst.sh</inst_hook>
```
Comment (theucke) [inline release_bp/OPX_dell_base_stretch.xml]: Is ztd not supported on Jessie?

File: inst-hooks/03-disable-ztd-file.pre-upgrade.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Ensure that we don't enter ZTD mode after an upgrade
+
+# Deleting the postinst hook ensures that even a failure in a later hook
+# would not leave the ZTD file hanging around on the standby filesystem.
+rm -f /alt/root/hooks/95-enable-ztd-mode.postinst.sh
```
Comment (heucke) [inline inst-hooks/03-disable-ztd-file.pre-upgrade.sh]: Is this directory created when ztd runs? Can you leave a comment regarding that? I don't see any `/alt` directory in the rootfs.

**Comment by skg-net** [comment]
Pull-request updated, HEAD is now 47702d4f8f008e2b1bd287bdaacde1df520d62f5

**Comment by skg-net** [comment]
@theucke Currently OPX doesn't support ZTD, we are adding the support. The one of the ONIE script is enabling that via config file. Addressed your other comments. Please take a look at it.

**Review by heucke — DISMISSED** [review]

**Comment by heucke** [comment]
Use squash and merge to combine your commit messages into one nice one.

**Comment by skg-net** [comment]
Pull-request updated, HEAD is now b349ac98c33e4c8d2dcd3da909405b70ea035a39

**Review by heucke — DISMISSED** [review]

**Comment by heucke** [comment]
Are you planning on upgrading to the new rootfs later? https://github.com/open-switch/opx-onie-installer/blob/master/release_bp/rootfs_stretch.xml#L2

**Comment by skg-net** [comment]
Pull-request updated, HEAD is now a962a09be9a0fdd2de227e08fbae7c4fd52fd1a3

**Comment by skg-net** [comment]
Pull-request updated, HEAD is now 87a38d8afb0c154513ad9d3cb429948193491391

**Comment by skg-net** [comment]
@theucke Updated to use the new rootfs.

**Review by heucke — APPROVED** [review]

**Pull request merged**